### PR TITLE
Add Options component to CRUD wp_options

### PR DESCRIPTION
**Pull request by moon0326**
This PR adds `Options` component.

1. It lists the last 30 options (This will be configurable in future versions).
2. it let you search for an option and delete it.

### Screenshots

Screen Shot 2021-03-03 at 5 58 04 PM

### Detailed test instructions

1. Checkout the branch
2. Start npm `npm start`
3. Activate the plugin and navigate to Tools -> WCA Test Helper

### CODE REVIEW

**Comment by becdetat** [comment]
It took some time for one of my searches to load - at least 10 seconds, enough time for me to look for the API call, switch to another tab, and switch back. Maybe a loading indicator would be a good future improvement? It seems pretty snappy now though so it's not a problem, I think it was just an outlier.

File: woocommerce-admin-test-helper.php [hunk 1]
```diff
@@ -50,10 +50,14 @@ function add_extension_register_script() {
 		),
 		$css_file_version
 	);
+
 	wp_enqueue_style( 'woocommerce-admin-test-helper' );
+
+    wp_enqueue_style('woocommerce-styles', '/wp-content/plugins/woocommerce-admin/dist/components/style.css');
```
Comment (becdetat) [inline woocommerce-admin-test-helper.php:56]: This won't work if the plugin isn't installed (if it is just running against the bundled version within WC core).

**Review by becdetat — APPROVED** [review]
Apart from the issue with WooCommerce Admin styles without the plugin installed, this looks great. Pre-approving.

File: woocommerce-admin-test-helper.php (hunk 1 above)
Comment (moon0326) [reply woocommerce-admin-test-helper.php:56]: That's my mistake. I included it temporarily when I was testing WC Admin component. I will take it out.

**Comment by moon0326** [comment]
> It took some time for one of my searches to load - at least 10 seconds, enough time for me to look for the API call, switch to another tab, and switch back. Maybe a loading indicator would be a good future improvement? It seems pretty snappy now though so it's not a problem, I think it was just an outlier.

Yup, that's on my todo list 👍

**Pull request merged**